ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__s3-bucket-cleanup.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  aws-client:
+    name: "4.12"
+    namespace: ocp
+    tag: upi-installer
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
```

Comment:
Can you please let me know, is this a correct requirement of memory or a bare minimum?

Reply:
We can change it, it seems to work fine, though. This is more of a guess than anything, and any minor changes probably won't make a difference as far as cost goes.